Download Populacao CSV.py

```diff
@@ -0,0 +1,62 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+import time
+chrome_options = Options()
+#chrome_options.add_argument('--headless')
+chrome_options.add_argument('--no-sandbox')
+chrome_options.add_argument('--disable-dev-shm-usage')
+chrome_options.add_experimental_option("prefs", {
+    "download.default_directory": "/path/to/download/directory",

```

Comment:
esse path to download directory é uma referencia padrão, não é? teria q ter uma referencia ao diretório em si que estraria se referindo

Reply:
eu n sei se entendi esse comentário. oq vc quer dizer com uma "referência ao diretório em si"? esse caminho coloca o arquivo na pasta de downloads, oq eu achei melhor já que eu n sei em que pasta específica o programa vai estar rodando em outros computadores. em teoria tanto o código quanto o arquivo csv ficariam na msm pasta. enfim desculpa se eu entendi errado a pergunta